src/axolotl/utils/schemas/config.py

```diff
@@ -220,6 +220,10 @@ class AxolotlInputConfig(
         },
     )
 
+    attention: AttentionBackend = Field(
+        default=AttentionBackend.flash,
```

Comment:
Not sure whether to default to enabling Flash? Some models don't work with flash (maybe Pixtral)